README.md

```diff
@@ -8,46 +8,58 @@ Codebase for Sparkle, brought to you by Sparkle.
 
 Clone the repo and cd into it
 
-```
+```bash
 git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`)
+<!-- Now, obtain the `.env` file for the environemnt (eg. staging) and save it (eg. `.env.staging`) -->
 
-```
+Now, obtain the `.env` file for the environment (ask someone on the engineering team) and save it.
+
+<!-- ```
 # copy in the .env files
 # symlink the staging .env file
 ln -s .env.staging.local .env
-```
+``` -->
 
-Now you're ready to start the server
+Install the npm modules
 
-```
+```bash
 npm i
+```
+
+Now you're ready to start the server! ✨
+
+```bash
 npm test
 npm start
```

Comment:
Technically I think that we need to be emulating the firebase functions locally (as below) before the server can properly start/the code work properly. Not 100% about this. It may only be required when we don't want to use the _actual_ staging functions (eg. when developing changes to the functions)

Not sure of the best way to fit that knowledge into this README (or if it's even the right place to). Let's see what you come up with :)

Reply:
So would someone need to do the firebase steps before the frontend ones?

Reply:
I did the steps in the order that they came in originally and everything seems to work fine (I think)

Reply:
> So would someone need to do the firebase steps before the frontend ones?

That was sort of what I was pondering/wondering when I asked my above question.

> I did the steps in the order that they came in originally and everything seems to work fine (I think)

Cool, if it worked that way, then probably should be fine as is :)